.sample_configs/variants.yaml

```diff
@@ -175,7 +177,7 @@ list_hyperparameter_tuning_jobs:
 list_model_evaluation_slices:
 - ''
 list_model_evaluations:
-- ''
+- tabular_forecasting
```

Comment:
this would replace the list_model_evaluation sample.  Please keep the "default" variant keyed with the empty string.

Reply:
Also - perhaps we don't need a special `list_model_evaluations` variant unless the call to `list_model_evaluations` is different in the `tabular_forecasting` use case.

Reply:
Oh I see. Good catch.